[2023-08-28T15:47:07.329+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_available manual__2023-08-28T15:47:04.991913+00:00 [queued]>
[2023-08-28T15:47:07.337+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_available manual__2023-08-28T15:47:04.991913+00:00 [queued]>
[2023-08-28T15:47:07.337+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-08-28T15:47:07.338+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-08-28T15:47:07.338+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-08-28T15:47:07.351+0000] {taskinstance.py:1383} INFO - Executing <Task(HttpSensor): is_api_available> on 2023-08-28 15:47:04.991913+00:00
[2023-08-28T15:47:07.356+0000] {standard_task_runner.py:55} INFO - Started process 3143 to run task
[2023-08-28T15:47:07.359+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'is_api_available', 'manual__2023-08-28T15:47:04.991913+00:00', '--job-id', '73', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmp4q1z976t']
[2023-08-28T15:47:07.361+0000] {standard_task_runner.py:83} INFO - Job 73: Subtask is_api_available
[2023-08-28T15:47:07.469+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.is_api_available manual__2023-08-28T15:47:04.991913+00:00 [running]> on host 22e2fb7a3f74
[2023-08-28T15:47:07.522+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=is_api_available
AIRFLOW_CTX_EXECUTION_DATE=2023-08-28T15:47:04.991913+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-08-28T15:47:04.991913+00:00
[2023-08-28T15:47:07.522+0000] {http.py:120} INFO - Poking: api/
[2023-08-28T15:47:07.529+0000] {base.py:71} INFO - Using connection ID 'user_api' for task execution.
[2023-08-28T15:47:07.531+0000] {http.py:148} INFO - Sending 'GET' to url: https://randomuser.me/api/
[2023-08-28T15:47:07.691+0000] {base.py:213} INFO - Success criteria met. Exiting.
[2023-08-28T15:47:07.699+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=is_api_available, execution_date=20230828T154704, start_date=20230828T154707, end_date=20230828T154707
[2023-08-28T15:47:07.732+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-08-28T15:47:07.761+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
